carbonmark/components/pages/Blog/Post/index.tsx

```diff
@@ -40,9 +40,11 @@ export const PostPage = (props: PostProps) => {
     );
   }
 
-  const publishedDate = `Published ${new Date(
-    props.post.publishedAt
-  ).toDateString()}`;
+  const publishedDate = `Published ${new Date(props.post.publishedAt)
+    .toDateString()
+    .split(" ")
+    .slice(1)
+    .join(" ")}`;
```

Comment:
As multiple languages need to be supported and Dates are formatted differently depending on each language
=> let's rather use `Intl.DateFormat` (see [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat))

So in this case this would be:

```TS

import { useRouter } from "next/router";

...
const { locale } = useRouter();

const publishedDate = new Date(props.post.publishedAt);

const formattedDate = new Intl.DateTimeFormat(locale, {
      day: "numeric",
      month: "long",
      year: "numeric",
   }).format(publishedDate);
```
Results for english => May 30, 2023
Results for german => 30. Mai 2023
Results for Hindi => 30 मई 2023

... and so on ...

Reply:
Of course, makes sense! Done